### DISCUSSION TITLE
Headless component combobox with @entangle value

### DISCUSSION THREAD

**Discussion by m15studio** [alpinejs/alpine#3739] [7. Bugs (archived)]
Hi there,
I'm using a combobox inside a livewire component.
On livewire component I have $customer_id and I bind it with alpine value using @entangle('customer_id').live

When I clic on combobox, instead to send to livewire only one single Ajax request with value, alpine component send continuously Ajax requests to update component property $customer_id

```
<div x-data="{
selected:  @entangle('customer_id').live,
frameworks: [
{
id: 1,
name: 'Laravel',
disabled: false,
},
{
id: 2,
name: 'Ruby on Rails',
disabled: false,
},
{
id: 3,
name: 'Phoenix',
disabled: true,
},
]}"></div>
```

**Comment by m15studio** [comment]
If I use 
`value:  @entangle('customer_id').live,`
inside Choices component (https://alpinejs.dev/component/choices) I haven't problems

**Comment by ekwoka** [comment]
This is a livewire issue.

**Reply by m15studio** [reply]
I'm sorry
I move issue to livewire discussions
